monkey/monkey_island/cc/next_ui/src/app/(auth)/login/page.tsx

```diff
@@ -79,23 +76,21 @@ const LoginPage = () => {
 
     const renderFormCard = () => {
         return (
-            <Card id={classes.card} variant="outlined" color="secondary">
+            <Card variant="outlined" sx={cardStyle(theme)}>
+                <MonkeyLogo sx={{ height: '50px' }} />
                 <Box component="form" onSubmit={handleSubmit} sx={{ mt: 1 }}>
-                    <Typography component="h1" variant="h5">
-                        Login
-                    </Typography>
                     <TextField
                         margin="normal"
                         required
                         fullWidth
                         id="username"
-                        label="username"
+                        label="Username"
                         name="username"
                         autoComplete="username"
                         value={loginFormValues.username}
                         onChange={handleLoginFormValueChange}
                         autoFocus
-                        color="secondary"
+                        style={{ background: theme.palette.background.default }}
```

Comment:
This seems like an appropriate use of inline style to me, especially since it is linked to the theme, which is an internal value

Reply:
Could be simplified though